WordPress/src/main/java/org/wordpress/android/models/discover/RecommendedBlog.kt

```diff
@@ -0,0 +1,11 @@
+package org.wordpress.android.models.discover
+
+data class RecommendedBlog(
```

Comment:
previous conversation https://github.com/AdamGrzybkowski/WordPress-Android/pull/1#discussion_r487567023

Reply:
Thank you for linking original discussion :). 

Some more thoughts:

> We would still need to store some information about the recommended_blogs card though.

We can continue to save simplified json into `tbl-discover-cards` similar to the format used for posts. Something like this: 
`{"type":"recommended_blogs","data":[{"blog_id":11184476,"feed_id":24204169},…}`

AFAIU` tbl-discover-cards` is used only to save simplified json with info enough to identify a record in the DB. If you notice actual fields for posts are saved into a table. I think this was done with the same  “single source of truth” principle in mind.

You’re right about using a single source of truth (`tbl-blog-info`) for the follow state. I’m bit worried that by storing blog fields like `name`, `description`, `icon` fields as json into `tbl-discover-cards`, we don’t violate the same principle. Lmkwyt.

As suggested, we can ask to include a blog model in the cards API and use the same flow as for posts. This can bring consistency in the code and it might be easier to understand in the future. 

> and we are fine with storing a lot of blogs over time

I think we’ll be storing lot of recommended blogs json data as well over time from the discover cards API into `tbl-discover-cards`. Wdyt?

Reply:
> I think we’ll be storing lot of recommended blogs json data as well over time from the discover cards API into tbl-discover-cards. Wdyt?

hmm tbh I assumed it's cleaned when refresh is done, but haven't confirmed it in the code, so you might be right! :) I will check it later though 

Overall I agree with what you wrote :) 
I'm still not sure If I will be able to reuse current models. I mean If we get full blog object I will have to use `ReaderBlog` instead of `ReaderRecommendedBlog`, and it feels like the second one should be used to parse recommended blogs 🤔  so this could bring some confusion as well. 
One thing that comes to my mind that could be a good example this solution might not be future proof is if the design changed and we would have to show e.g. `score` we couldn't do that by using `ReaderBlog`. But maybe that's just going a bit too far?

Reply:
> I’m still not sure If I will be able to reuse current models. I mean If we get full blog object I will have to use `ReaderBlog` instead of `ReaderRecommendedBlog`

Using `ReaderBlog` is completely fine -> we can completely remove `ReaderRecommendedBlog` if we won’t use it - which is fine as well.  I think it is currently used on "Reader -> Following tab -> Filter -> Manage Sites and Tags -> Sites you may like" but might be removed after new changes. Let's not worry about it now.

> One thing that comes to my mind that could be a good example this solution might not be future proof is if the design changed and we would have to show e.g. `score` we couldn’t do that by using `ReaderBlog`. But maybe that's just going a bit too far?

I agree that it can happen, but it’s not something we expect - we’ll have to deal with it. Again, let's not worry about it now :).

Reply:
ok! Thanks for all the answers and discussion @ashiagr  :) 
I will ask for a full blog object